ddtrace/appsec/_iast/_taint_tracking/Aspects/Helpers.cpp

```diff
@@ -60,7 +60,7 @@ as_formatted_evidence(const string& text,
 
     for (const auto& taint_range : text_ranges) {
         string content;
-        if (!tag_mapping_mode or tag_mapping_mode.value() == TagMappingMode::Normal) {
+        if (!tag_mapping_mode or *tag_mapping_mode == TagMappingMode::Normal) {
```

Comment:
You have to check with `has_value` before the dereferencing of an optional. It's undefined behaviour otherwise. Same for the others.